docs/javadoc/resource-files/stylesheet.css

```
@@ -0,0 +1,1667 @@
+/*
+ * Copyright (c) 2010, 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+ */
+
+/*
+ * Javadoc style sheet
+ */
+
+@import url('fonts/dejavu.css');
```

Stylelint (17.4.0)

[error] Expected "url('fonts/dejavu.css')" to be "'fonts/dejavu.css'" (import-notation)

(import-notation)